include/csc232.h

```diff
@@ -59,29 +59,74 @@ using std::setw;
 
 // TODO: 1.1 Declare a named constant ROSTER_SIZE initialized to the integer value 10.
```

Comment:
STYLE: Once you complete a `TODO:`, you should delete the lines that specify the `TODO`. This comment applies throughout this lab (i.e., I'm not going to paste it again all over the place).